pages/index.tsx

```diff
@@ -67,15 +66,23 @@ const ImageList = styled.ul`
   width: 100%;
 
   li {
+    border: 3px solid;
+    border-width: min(6vh,8vw);
+    border-image-source: url(frame.svg);
+    border-image-slice: 12%;
     margin: 1rem 0;
-    width: 100%;
+    width: 95%;
 
     ${(props) => props.theme.screenSizes.tabletPortraitPlus} {
-      width: calc(100% / 2);
+      width: calc(95% / 2);
     }
 
     ${(props) => props.theme.screenSizes.tabletLandscapePlus} {
-      width: calc(100% / 3);
+      width: calc(95% / 3);
+    }
```

Comment:
These are ratios so would avoid adjusting by a bit - it's not really clear what 5% is. I can fix with the labels.

Reply:
I'll merge this in and add to #46  to do this. I'm not sure what "fix with labels" is.